package/kernel/linux/modules/other.mk

```diff
@@ -1225,3 +1225,18 @@ define KernelPackage/it87-wdt/description
 endef
 
 $(eval $(call KernelPackage,it87-wdt))
+
+
+define KernelPackage/pinctrl-sx150x
```

Comment:
If this has to be built-in, why not just enable it in subtarget kernel config instead of making it a package?

Reply:
It could be because it only depends on i2c. I will rework this without this patch.